docs/mkdocs.yml

```diff
@@ -0,0 +1,67 @@
+# Meta parameters for mkdocs
+docs_dir: src/
+
+# Project information
+site_name: "Simpleflow documentation"
+copyright: "Copyright &copy; 2017 - Botify"
```

Comment:
The project is licensed as MIT (which is my goto license personally), and MIT requires a copyright section at the top of the license. Not sure the docs website has to be explicitly licensed the same way though ¯\_(⊙︿⊙)_/¯